.circleci/config.yml

```diff
@@ -69,11 +72,8 @@ commands:
   build_and_run_xcode_tests:
     steps:
       - run:
-          command: xcodebuild clean build-for-testing -project "Apollo.xcodeproj" -scheme "${CIRCLE_XCODE_SCHEME}" -destination "${DESTINATION}" -testPlan "${CIRCLE_XCODE_TEST_PLAN}" | xcbeautify
-          name: Clean and build for testing
-      - run:
-          command: xcodebuild test-without-building -resultBundlePath ~/TestResults/ResultBundle.xcresult -project "Apollo.xcodeproj" -scheme "${CIRCLE_XCODE_SCHEME}" -destination "${DESTINATION}" -testPlan "${CIRCLE_XCODE_TEST_PLAN}" | xcbeautify
-          name: Run Xcode tests
+          command: xcodebuild clean test -resultBundlePath ~/TestResults/ResultBundle.xcresult -project "Apollo.xcodeproj" -scheme "${CIRCLE_XCODE_SCHEME}" -destination "${DESTINATION}" -testPlan "${CIRCLE_XCODE_TEST_PLAN}" | xcbeautify
```

Comment:
This condenses the two steps into one. I strongly suspect this is yet [another Xcode `14.3` bug](https://stackoverflow.com/questions/75979633/xcodebuild-ignores-testplan-when-using-test-without-building).

Besides that though there really is no reason we need two steps for this, we're not sharing the build artifact into any other step/job, and the single step actually saves time on test runs.